extensions/pkg/controller/extension/actuator.go

```diff
@@ -28,6 +28,8 @@ type Actuator interface {
 	Reconcile(context.Context, logr.Logger, *extensionsv1alpha1.Extension) error
 	// Delete the Extension resource.
 	Delete(context.Context, logr.Logger, *extensionsv1alpha1.Extension) error
+	// ForceDelete force deletes the Extension resource.
```

Comment:
Serialize the doc string to to have same pattern every where/
```suggestion
	// ForceDelete forcefully deletes the Extension resource.
```